modules/ROOT/pages/dataweave-cookbook-define-a-custom-addition-function0.adoc

```diff
@@ -0,0 +1,135 @@
+= Define a Custom Addition Function
```

Comment:
Need to remove this page. Looks like a copy of the preceding file but has a 0 at the end of the file name base. 
Compare
modules/ROOT/pages/dataweave-cookbook-define-a-custom-addition-function.adoc
to
modules/ROOT/pages/dataweave-cookbook-define-a-custom-addition-function0.adoc

Reply:
handled via 9782643a22c97f93c5c46ae5df4e9bd9c24e8d0b